examples/triangle.py

```diff
@@ -143,11 +145,9 @@ def draw_frame():
     return device
 
 
-if __name__ == "__main__":
+canvas = WgpuCanvas(size=(640, 480), title="wgpu triangle")
+main(canvas)
```

Comment:
I think this may break the tringle_xx.py examples.

Reply:
Right... Will have another look.

Reply:
Reverted. Only the cube example is tested now.